docs/Documentation/Drivers/Image/Params.md

```diff
@@ -1,2 +1,142 @@
+# VIC Model Parameters
+
 !!! Warning
     Docs not written yet.
+
+# Converting from VIC 4 to VIC 5
+
+We are developing a few scripts to aid users in moving from VIC 4 to VIC 5.  These scripts are currently distributed via [`tonic`](https://github.com/UW-Hydro/tonic).
+
+### Installing Tonic Using Anaconda
+
+The easiest way to install Tonic and its dependencies is to use the [Anaconda Python distribution](https://store.continuum.io/cshop/anaconda/).
+
+To install Anaconda, follow these two simple steps (check to make sure the installer version is the most current)
+
+1.  download and run the Anaconda installer:  [http://continuum.io/downloads](http://continuum.io/downloads)
+
+2.  setup a virtual environment for Tonic
+```shell
+conda create -n tonic anaconda
+source activate tonic
+```
+
+*Note:  you'll need to do the `source activate tonic` to activate the Tonic virtual environment in any new shells.*
+
+Now, download the Tonic source code:
+
+```shell
+git clone git@github.com:UW-Hydro/tonic.git
+```
+
+From the Tonic source code repository, Tonic can be installed using Python's `distutils`:
+
+```shell
+python setup.py install
+```
+
+This installs a top level script, `tonic`, into your bin/ directory and the `tonic` package into your Python path.
+
+If you don't want to use the Anaconda installation I've shown above, you can build the package in your local python installation using:
+```python
+python setup.py develop
+```
+
+### Option 2a:  Using a local Python Install (With Write Permissions)
+
+If you have write permissions to the location of your Python distribution, you can just run
+
+```shell
+python setup.py install
+```
+
+from the top level Tonic directory.  This will install Tonic into your `$PYTHONPATH`.
+
+### Option 2b:  Using a local Python Install (Without Write Permissions)
+
+If you do not have write permissions, you can install Tonic in your local `$PYTHONPATH` by following these steps:
+
+Create a `lib/python` directory in your `$HOME` directory:
+
+```shell
+mkdir -p $HOME/lib/python/
+```
+
+Add this library path to your `$PYTHONPATH` in your `.bashrc`:
+
+```shell
+export PYTHONPATH=$HOME/lib/python:$PYTHONPATH
+```
+
+Run `setup.py`:
+
+```shell
+python setup.py install --home=$HOME
+```
+
+### Testing your install
+
+From the following command line:
+
+```shell
+vic_utils -h
+python -c 'import tonic'
+```
+
+If you don't get any errors, you should be ready to use `tonic`.
+
```

Comment:
So, I had an issue with netCDF4 not being found.  Perhaps the reader should check whether they have a working netCDF4 installation and/or that Anaconda is looking in the right place for it, before getting this far.